examples/remote/binary_dependencies/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_rust//rust:rust.bzl", "rust_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+rust_binary(
+    name = "binary_dependencies_bin",
+    srcs = ["src/main.rs"],
+    data = [
+        "//remote/binary_dependencies/cargo:cargo_bin_wasm_bindgen",
```

Comment:
Oh actually I do have a comment.

Are there any binary deps with less transitive dependencies? That might cut down on the size of this in the repo

Reply:
I can do some digging but this represents the very thing I'd like to address in `rules_rust`. Might I be able to do this in a future PR?

Reply:
Or do you know of another create that exhibits this behavior? (I'll try to look tonight either way)

Reply:
Found one with 100 less transitive dependencies